packages/powersync/lib/src/stream_utils.dart

```diff
@@ -68,6 +68,13 @@ Stream<Object?> ndjson(ByteStream input) {
   return jsonInput;
 }
 
+/// Given a raw ByteStream, parse each line as JSON.
+Stream<String> newlines(ByteStream input) {
```

Comment:
I think this isn't actually used anywhere.

Reply:
Removed